akka-javasdk/src/main/scala/akka/javasdk/impl/SdkRunner.scala

```diff
@@ -756,6 +756,11 @@ private final class Sdk(
         Some(
           wiredInstance[ServiceSetup](serviceClassClass.asInstanceOf[Class[ServiceSetup]])(
             sideEffectingComponentInjects(None)))
+
+      case Some(serviceClassClass) =>
+        //just wiring the class
+        wiredInstance[Any](serviceClassClass.asInstanceOf[Class[Any]])(sideEffectingComponentInjects(None))
```

Comment:
@octonato this a fix for: https://github.com/lightbend/kalix-runtime/issues/4051

Reply:
Can you move it to its own PR? 

Better not to sneak this fix in a PR about model config. 

Also, it's not clear if that's the fix we should have. 
And is that impl here correct? How is that even compiling? 

The type is `Option[ServiceSetup]`, but we can write a `Any`!